todo_app/app.py

```diff
@@ -1,15 +1,31 @@
-from flask import Flask
+from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import *
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
 
+@app.route('/addItem', methods = ['POST'])
+def addItem():
```

Comment:
Python naming convention is snake_case for function and variable names. So `add_item`.

Reply:
Done